pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java

```diff
@@ -449,7 +449,8 @@ private class GetRetention extends CliCommand {
         private boolean applied = false;
 
         @Parameter(names = { "--global", "-g" }, description = "Whether to get this policy globally. "
-                + "If set to true, broker returned global topic policies")
+                + "If set to true, broker returned global topic policies"
+                + "If set to false or not set,the broker returned local topic policies")
```

Comment:
```suggestion
                + "If set to true, the broker returns global topic policies"
                + "If set to false or not set, the broker returns local topic policies")
```

Reply:
write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true

https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp